src/syntax/trees/ParseTree.js

```diff
@@ -238,7 +243,13 @@ export class ParseTree {
   }
 
   isStatementListItem() {
-    return this.isStatement() || this.isDeclaration();
+    return this.isStatement() || this.isDeclaration() ||
+        // TODO(arv): When transforming modules we can get a type alias. Once
+        // #1995 is fixed we can change the order of these transformers and the
+        // type alias will get removed before it gets inserted into an invalid
+        // location.
+        // https://github.com/google/traceur-compiler/issues/1995
```

Comment:
not big deal, but type-alias here would help disambiguate type in `this.type` vs type in `TYPE_ALIAS_DECLARATION`